src/test/resources/features/userprofile/edit_user_profile.feature

```diff
@@ -0,0 +1,17 @@
+@UserProfile
+Feature: Edit User Profile
+
+  @ResetUserInformation @After
+  Scenario: Edit User's information in My Profile section
+    Given I log in to Pivotal with Editable User credentials
+    When I navigate to PROFILE_PAGE
+      And I edit My Profile with the following information
+        | User Name | UpdatedUsernameUNIQUE_ID |
+        | Name      | Updated name UNIQUE_ID   |
+        | Initials  | UU                       |
+    Then "Changed saved" message should be displayed in My Profile section
+      And the user information should be updated in My Profile section
+      And my Name should be updated in the User Management Menu
```

Comment:
In some step you are using my Name and in other the user inforamtion, it is confusing